app/src/main/java/com/github/se/eventradar/model/repository/message/FirebaseMessageRepository.kt

```diff
@@ -65,11 +66,12 @@ class FirebaseMessageRepository(db: FirebaseFirestore = Firebase.firestore) : IM
             .await()
 
     if (resultDocument == null || resultDocument.isEmpty) {
-      return createNewMessageHistory(user1, user2)
+      return Resource.Failure(Exception("No message history found between users"))
```

Comment:
Question: I believe what we had before worked fine and now there is an inconsistency with `getMessages(uid: String)`. Can you explain your reasoning?

Reply:
After discussing with @christinehahaa , the reasoning is that we shouldn't create a history for users that have not yet texted each other.